src/ore/src/tracing.rs

```diff
@@ -309,6 +311,17 @@ impl OpenTelemetryContext {
         tracing::Span::current().set_parent(parent_cx);
     }
 
+    /// Attaches this `Context` to the given span,
+    /// as a link.
+    ///
+    /// If there is not enough information in this `OpenTelemetryContext`
+    /// to create a context, then the current thread's `Context` is used
+    /// defaulting to the default `Context`.
+    pub fn attach_as_link_to(&self, span: &tracing::Span) {
+        let cx = global::get_text_map_propagator(|prop| prop.extract(&self.inner));
+        span.add_link(cx.span().span_context().clone());
+    }
```

Comment:
Does [`Span::follows_from`](https://docs.rs/tracing/latest/tracing/struct.Span.html#method.follows_from) not do what we need?

Reply:
I believe its functionally the same